WordPress/src/main/res/values/strings.xml

```diff
@@ -665,7 +665,7 @@
     <string name="reader_hint_reblog_comment">Add your thoughts (optional)</string>
 
     <!-- TextView labels -->
-    <string name="reader_label_new_posts">Show new posts</string>
+    <string name="reader_label_new_posts">&#8593; New posts</string>
```

Comment:
The up arrow looks a little weird (thin and too high imo), could we use a `drawableLeft` with a proper icon?